core/pom.xml

```diff
@@ -282,5 +277,50 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>hadoop2</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
```

Comment:
These still have the activeByDefault flag, which must be removed, because it does not do what you think. :)